### PR TITLE
Add all files

### PR DESCRIPTION
**Pull request by flavios98**

### CODE REVIEW

File: .nuspec [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
+    <metadata>
+        <id>Messerli.Utility</id>
```
Comment (dngroth) [inline .nuspec]: User correct name `Messerli.FileOpeningBuilder`

File: .nuspec [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
+    <metadata>
+        <id>Messerli.Utility</id>
+        <version>0.1.1</version>
```
Comment (dngroth) [inline .nuspec]: Remove version tag or make it 0.0.0

File: .nuspec [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
+    <metadata>
+        <id>Messerli.Utility</id>
+        <version>0.1.1</version>
+        <authors>Jan Nils Ferner, Dominic Groth, Thomas Bruderer</authors>
```
Comment (dngroth) [inline .nuspec]: Add yourself to author list if you want?

File: .nuspec [hunk 4]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
+    <metadata>
+        <id>Messerli.Utility</id>
+        <version>0.1.1</version>
+        <authors>Jan Nils Ferner, Dominic Groth, Thomas Bruderer</authors>
+        <owners>messerli-informatik-ag</owners>
+        <projectUrl>http://github.com/messerli-informatik-ag/utility</projectUrl>
+        <license type="expression">MIT or Apache-2.0</license>
+        <description>Various utility functions and extension methods used by other Messerli libraries.</description>
```
Comment (dngroth) [inline .nuspec]: Add matching description

File: .nuspec [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+    <metadata>
+        <id>Messerli.Utility</id>
+        <version>0.1.1</version>
+        <authors>Jan Nils Ferner, Dominic Groth, Thomas Bruderer</authors>
+        <owners>messerli-informatik-ag</owners>
+        <projectUrl>http://github.com/messerli-informatik-ag/utility</projectUrl>
+        <license type="expression">MIT or Apache-2.0</license>
+        <description>Various utility functions and extension methods used by other Messerli libraries.</description>
+        <copyright>Copyright ©2019 Messerli Informatik Ag</copyright>
+        <tags>utility math</tags>
```
Comment (dngroth) [inline .nuspec]: Add matching tags

File: paket.dependencies [hunk 6]
```diff
@@ -0,0 +1,12 @@
+framework auto-detect
+
+source https://www.nuget.org/api/v2
+
+nuget Autofac ~> 4.8
```
Comment (dngroth) [inline paket.dependencies]: Does this project need all this packets?

File: paket.dependencies [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+source https://www.nuget.org/api/v2
+
+nuget Autofac ~> 4.8
+nuget Equals.Fody ~> 1.9
+nuget Fody ~> 4.0
+nuget JetBrains.Annotations ~> 2018.2
+nuget Messerli.Test.Utility ~> 0.1.0
+nuget Messerli.Utility ~> 0.1.1
+nuget Moq ~> 4.10
+nuget Newtonsoft.Json ~> 12.0
```
Comment (dngroth) [inline paket.dependencies]: Add terminating newline

File: readme.md [hunk 8]
```diff
@@ -0,0 +1,6 @@
+# Utility
```
Comment (dngroth) [inline readme.md]: Correct readme use same message as in `.nuspec`
Adjust badge links

**Review by dngroth — CHANGES_REQUESTED** [review]

File: paket.dependencies (hunk 6 above)
Comment (flavios98) [reply paket.dependencies]: ja

File: paket.dependencies (hunk 7 above)
Comment (flavios98) [reply paket.dependencies]: https://github.com/messerli-informatik-ag/file-opening-builder/pull/1/commits/5616f8c6372e50e2780fc147eaba1c674f2a365a

File: .nuspec (hunk 4 above)
Comment (flavios98) [reply .nuspec]: 0bd785ad230e676bb177170ab1ec274a93336b64

File: .nuspec (hunk 5 above)
Comment (flavios98) [reply .nuspec]: a0584de71a3a0e535518b43ef4d0a7b2bec2d577

File: .nuspec (hunk 3 above)
Comment (flavios98) [reply .nuspec]: a0584de71a3a0e535518b43ef4d0a7b2bec2d577

File: changelog.md [hunk 9]
```diff
@@ -0,0 +1,7 @@
+# Changelog
```
Comment (dngroth) [inline changelog.md:1]: Adjust chnagelog to
```md
# Changelog

## Unreleased
- Initial release
```

**Review by dngroth — CHANGES_REQUESTED** [review]

File: .nuspec (hunk 2 above)
Comment (flavios98) [reply .nuspec]: 54b0258c00e1b4e582cece5fdbc542789c37f36b

Comment (flavios98) [reply .nuspec]: a0584de

File: .nuspec (hunk 1 above)
Comment (flavios98) [reply .nuspec]: a0584de

File: readme.md (hunk 8 above)
Comment (flavios98) [reply readme.md]: 57ad565df9c4af4bbf2bdbedef4fc7418872b95b

File: changelog.md (hunk 9 above)
Comment (flavios98) [reply changelog.md:1]: 5ad9051d3e49e58b75cf355556b06722e662d5a1

Comment (dngroth) [reply changelog.md:1]: Add
```md
## Unreleased
- Initial release
```

File: paket.dependencies (hunk 6 above)
Comment (dngroth) [reply paket.dependencies]: I can only see following dependencies
`FileOpeningBuilder\paket.references` does not need any pakets
`FileOpeningBuilder.Test\paket.references` does only need `using Messerli.Test.Utility`

Comment (flavios98) [reply paket.dependencies]: image

image

File: readme.md (hunk 8 above)
Comment (dngroth) [reply readme.md]: Restore Badges just adjust the links in them

File: paket.dependencies (hunk 6 above)
Comment (dngroth) [reply paket.dependencies]: True but those files souldn't contain all of the pakets E.g autofac or JetBrains.Annotations

File: changelog.md (hunk 9 above)
Comment (flavios98) [reply changelog.md:1]: 4427c81efb647664171d78896ba2417c6ba505b2

File: readme.md (hunk 8 above)
Comment (flavios98) [reply readme.md]: 3aa7d0a599c9a4c43e976e29fb475841a5cfb335

File: paket.dependencies (hunk 6 above)
Comment (flavios98) [reply paket.dependencies]: ac89b63d2e16f9159c606ccbb9896015b6bccc8b

File: readme.md (hunk 8 above)
Comment (dngroth) [reply readme.md]: Remove json paket

Comment (flavios98) [reply readme.md]: 8b262e614b970440bcf7cd897f605ec26ac484ce comment should be by paket.dependencies

**Review by dngroth — APPROVED** [review]

**Pull request merged**